### ISSUE TITLE
Are we open to changing the test file names?

### ISSUE THREAD

**Issue by binomialstew** [storybooks/babel-plugin-react-docgen#55]
When working on the last PR, I noticed that the `test/fixtures` file names were not as clear as I would like. For example, the test source files are named `actual.js`. I would expect these to be named `source.js` and any generated output to be named `actual.js`. `expected.js` would stay the same. This would also allow for an `npm` command that generates `actual.js` without overwriting the existing file—enabling easier comparison between the two. If this is something you are open to, I can create a PR—it is basically ready to go.

**Comment by danielduan** [comment]
I completely agree all of the above. Maybe even reorganizing the folders and folder names so that they make more sense in terms of grouping too.
